test/functional/wallet_listreceivedby.py

```diff
@@ -119,6 +119,9 @@ def run_test(self):
         # Trying to getreceivedby for an address the wallet doesn't own should return an error
         assert_raises_rpc_error(-4, "Address not found in wallet", self.nodes[0].getreceivedbyaddress, addr)
 
+        # Trying to getreceivedby for a label the wallet doesn't own should return an error
```

Comment:
- s/getreceivedby/getreceivedbylabel/
- suggested change to comment to be like the other logging output
-  either move the test further down into a getreceivedbylabel section or change the comment to a log

```suggestion
        self.log.info("getreceivedbylabel returns an error if the wallet doesn't own the label")
```